addon_toolkit/credentials.py

```diff
@@ -27,6 +27,29 @@ class AccessKeySecretKeyCredentials(Credentials):
     secret_key: str
 
 
+@dataclasses.dataclass(frozen=True, slots=True)
+class OAuth1TokenCredentials:
```

Comment:
I think for OAuth1 we should still use `AccessTokenCredentials`. The idea of the `AccessTokenCredentials` dataclass is to implement the `iter_headers` method so that `GravyvaletHttpRequestors` can automatically append the appropriate auth headers for the oauth requests.

Reply:
Depends on if OAuth1 requests require the secret to be sent along with the token. If so, it's a different credentials shape. Otherwise, yes, we should keep using AccessTokenCredentials and create an OAuth1 equivalent to `OAuth2TokenMetadata` for tracking other details

Reply:
Well, auth headers are different for OAuth1.0a and OAuth2, and even more: they are different for Zotero